lib/ansiblelint/formatters/__init__.py

```diff
@@ -87,6 +87,33 @@ def format(self, match, colored=False):
                                     match.message)
 
 
+class AnnotationsFormatter(BaseFormatter):
+    # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message
+    """Emits errors in format that is recognized by Github Annotation parsing.
+
+    ::warning file={name},line={line},col={col}::{message}
```

Comment:
How about mentioning all the possibilities?
```suggestion
    ::debug file={name},line={line},col={col}::{message}
    ::error file={name},line={line},col={col}::{message}
    ::warning file={name},line={line},col={col}::{message}
```